types/pikaday/index.d.ts

```diff
@@ -90,12 +90,12 @@ declare class Pikaday {
     /**
      * Update the minimum/earliest date that can be selected.
      */
-    setMinDate(date: Date): void;
+    setMinDate(date: Date | null): void;
```

Comment:
It could make sense to make the parameter optional instead of the `| null`.

Reply:
Well, looking at the implementation, the type could actually be `any` since anything other than a `Date` would clear the minimum date. But I think using `null` in the type makes it more explicit what the methods do.